config/content.d/developer.rackspace.com.json

```diff
@@ -23,7 +23,7 @@
           "/docs/cloud-networks/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-networks/",
           "/docs/cloud-big-data/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-big-data/v2/",
           "/docs/autoscale/v1/developer-guide/": "https://github.com/rackerlabs/otter/",
-          "/docs/cloud-servers/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-servers/",
+          "/docs/cloud-servers/v2/": "https://github.com/rackerlabs/docs-cloud-servers/v2/",
           "/docs/cloud-files/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-files/",
```

Comment:
That's the way the nexus configuration works. I'll add it to the file, but it's documented elsewhere.